poi-ooxml/src/test/java/org/apache/poi/xssf/streaming/TestOutOfOrderColumns.java

```diff
@@ -0,0 +1,112 @@
+package org.apache.poi.xssf.streaming;

```

Comment:
All POI files need Apache License headers - have a look at any Java file in the repo for an example.